build-config/scripts/onie-mk-iso.sh

```diff
@@ -154,9 +154,8 @@ if [ "$UEFI_ENABLE" = "yes" ] ; then
 
     # Generate UEFI format GRUB image
     mkdir -p $RECOVERY_EFI_BOOT_DIR
-    $GRUB_HOST_BIN_UEFI_DIR/grub-mkimage \
-        --format=x86_64-efi \
-        --directory=$GRUB_HOST_LIB_UEFI_DIR \
+    grub-mkimage \
```

Comment:
While running the grub-mkimage command in GRUB_HOST_BIN_UEFI_DIR for arm64-efi format, I was getting this error : "relocation 0x2 is not implemented yet". Which is why I used the grub-mkimage of build machine.